tests/test_core_fileproc.py

```diff
@@ -53,18 +53,18 @@ def test_check_file_type(file_path, file_types):
     ["tests/test_files/sample_test_file.md", "tests/test_files/sample_test_file.py"],
 )
 @pytest.mark.parametrize(
-    "white_list_patterns", [["[.py]"], ["[.md]"], ["tests/test_file"]]
+    "exclude_patterns", [["[.py]"], ["[.md]"], ["tests/test_file"]]
```

Comment:
```suggestion
    "exclude_patterns", [["[.]py$"], ["[.]md$"], ["tests/test_file"]]
```

Reply:
can you explain this change to me please?

Reply:
Sure! So your current regular expressions are matching exactly .py. We usually only need brackets when we want an exact match of a character that could be a regular expression (in the example above, the period). So I moved the brackets around just the periods, left the letters as is, and added a $ to indicate we only want to match the end of the line (e.g., we wouldn't want to match filename.python-bindings or `.mdl`